lib/mayaUsd/utils/utilFileSystem.h

```diff
@@ -49,6 +49,17 @@ std::string getMayaSceneFileDir();
 MAYAUSD_CORE_PUBLIC
 std::string getMayaWorkspaceScenesDir();
 
+/*! \brief takes in an absolute file path and returns the path relative to maya scene file
```

Comment:
Should this comment mention that when there is no scene file, the absolute (input) path will be returned.